.github/templates/free-space/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: free-space
```

Comment:
I think this just came from main branch. I added this to free space on ubuntu runners, because the previous one that we were using wasn't good enough.